packages/curve-ui-kit/src/shared/ui/PositionDetails/tooltips/LiqThresholdMetricTooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import { Stack, Typography } from '@mui/material'
+import { FORMAT_OPTIONS, formatNumber } from '@ui/utils/utilsFormat'
+import { t } from '@ui-kit/lib/i18n'
+import { BandRange, LiquidationRange } from '@ui-kit/shared/ui/PositionDetails'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+type LiqThresholdTooltipProps = {
+  liquidationRange: LiquidationRange | undefined | null
+  bandRange: BandRange | undefined | null
+}
+
+export const LiqThresholdTooltip = ({ liquidationRange, bandRange }: LiqThresholdTooltipProps) => (
```

Comment:
No reason to abbreviate IMO
```suggestion
export const LiquorThresholdTooltip = ({ liquidationRange, bandRange }: LiquorThresholdTooltipProps<🍺>) => (
```